bedrock/thunderbird/urls.py

```diff
@@ -31,20 +34,26 @@
 
     # Release-related pages
     url(r'^thunderbird/(?:%s/)?all/$' % channel_re,
-        views.all_downloads, name='thunderbird.all'),
+        views.all_downloads, name='thunderbird.latest.all'),
     url('^thunderbird/releases/$',
         bedrock.releasenotes.views.releases_index,
         {'product': 'Thunderbird'}, name='thunderbird.releases.index'),
     url('^thunderbird/(?:%s/)?notes/$' % channel_re,
         bedrock.releasenotes.views.latest_notes,
-        {'product': 'Thunderbird'}, name='thunderbird.notes'),
+        {'product': 'thunderbird'}, name='thunderbird.latest.notes'),
     url('^thunderbird/latest/releasenotes/$',
         bedrock.releasenotes.views.latest_notes,
         {'product': 'thunderbird', 'channel': 'release'}),
+    url(notes_re,
+        bedrock.releasenotes.views.release_notes,
+        {'product': 'Thunderbird'}, name='thunderbird.notes'),
     url('^thunderbird/(?:%s/)?system-requirements/$' % channel_re,
         bedrock.releasenotes.views.latest_sysreq,
-        {'product': 'Thunderbird'}, name='thunderbird.sysreq'),
+        {'product': 'thunderbird'}, name='thunderbird.latest.sysreq'),
     url('^thunderbird/latest/system-requirements/$',
         bedrock.releasenotes.views.latest_sysreq,
         {'product': 'thunderbird', 'channel': 'release'}),
+    url(sysreq_re,
+        bedrock.releasenotes.views.system_requirements,
+        {'product': 'Thunderbird'}, name='thunderbird.sysreq'),
```

Comment:
No, this one is upper-case somehow. I think everything should be lower case, but it likely requires change to Nucleus.